src/mediapipe_internal/pythonnoderesource.cpp

```diff
@@ -37,6 +37,32 @@ PythonNodeResource::PythonNodeResource() {
     this->nodeResourceObject = nullptr;
 }
 
+Status PythonNodeResource::finalize() {
+    if (this->nodeResourceObject) {
```

Comment:
Is it possible that in this part of execution nodeResourceObject would be nullptr?

Reply:
We should inform administrator since this is abnormal behavior